script/Systems/Overlay/HUD.lua

```diff
@@ -773,139 +776,312 @@ function HUD:drawTacticalMap (a)
   UI.DrawEx.Line(cx + 48, self.sy - 124, cx,      self.sy -  78, Config.ui.color.meterBar, false)
 end
 
+local updateTargetsInterval = 1 / 60
```

Comment:
just so u know, making this a config option in "part 2" once i figured where else we can improve.